tiktoken/model.py

```diff
@@ -26,6 +26,8 @@
     "babbage-002": "cl100k_base",
     # embeddings
     "text-embedding-ada-002": "cl100k_base",
+    "text-embedding-3-small": "cl100k_base",
+    "text-embedding-3-large": "cl100k_base",
```

Comment:
Is there any public documentation which clarifies that the new embedding models have cl100k_base encoding?

Reply:
It seems that they're clarified here: https://platform.openai.com/docs/guides/embeddings/embedding-models

Reply:
@zafercavdar they are mentioning it in their cookbook
Attaching a source https://github.com/openai/openai-cookbook/blob/main/examples/How_to_count_tokens_with_tiktoken.ipynb
<img width="958" alt="Screenshot 2024-02-01 at 5 11 33 AM" src="https://github.com/openai/tiktoken/assets/23660137/c3be7414-ec50-4bb4-b24a-7740fc99e8b2">

Reply:
@zafercavdar Can you approve the PR